deepsea.spec

```diff
@@ -45,16 +45,34 @@ BuildArch:      noarch
 A collection of Salt files providing a deployment of Ceph as a series of stages.
 
 
+#################################################################################
+# subpackages
+#################################################################################
+%package cli
+Summary:        DeepSea CLI
+Group:          Systems/Filesystems
+BuildRequires:  python-setuptools
+Requires:       deepsea = %{version}-%{release}
+Requires:       python-setuptools
+Requires:       python-click
```

Comment:
ah, ok, now I see this, but I don't see the `%package cli` and `%description cli` sections?

Reply:
argh, nevermind. I'm turning blind in my old age.